shared/settings/render.desktop.js

```diff
@@ -42,9 +81,10 @@ class Render extends Component<void, Props, State> {
       return this._renderComingSoon()
     }
 
-    return <MenuList items={this.state.menuItems} />
+    return <NavSettings
```

Comment:
@MarcoPolo we should discuss how we want to handle this routing as this is essentially an embedded tab router itself. we'll also have a similar thing in chat so we should have a generic way to do this. maybe this already exists?